### ISSUE TITLE
Pull request merging, fork unification

### ISSUE THREAD

**Issue by gacelita** [wuub/SublimeREPL#508]
I just made a pull request and I don't think it is ever going to be merged.
This is a pity for the ST3 community. It would be a good idea to join the efforts that are currently split between the forks, and create a new upstream for this project (where I can send my pull request for it to be merged).

**Comment by gacelita** [comment]
@tomschenkjr @fukamachi @emestee Sorry for bothering you, but your forks are the most popular ones, and I'd appreciate your comments here. Thank you

**Comment by gacelita** [comment]
@avamsi
@bofm
@brianhuey
@Christopher-Bui
@cmische
@danfrankj
@Demetrio92
@divinites
@edubkendo
@ehuss
@elstgav
@gniquil
@JanVanRyswyck
@jrdrg
@demidovakatya
@laughedelic
@ldmberman
@lemonlatte
@lionandoil
@mantkiew
@maxheld83
@mvoidex
@pablorusso
@rixth
@robmuh
@sergiovilar
@simonzack
@smiler
@swliuproxy
@tommireinikainen
@vulq

All of you have open pull requests. If you like the idea, we can create an organization, fork the repo, and merge *everything* there.

**Comment by divinites** [comment]
I have no objection against this idea.

**Comment by avamsi** [comment]
+1. It would be better if *this* repo can be transferred to an organization (**if** @wuub is willing to, of course)

**Comment by wuub** [comment]
I'm all for transferring this to an organization.

**Comment by gacelita** [comment]
Great, thank you, the organization has been created: https://github.com/SublimeREPL

**Comment by wuub** [comment]
@JoelSanchez please let me sleep on it for a day or two on how to do this w/o disrupting package control and  other projects that are potentially linked with this repository.

**Comment by lionandoil** [comment]
sounds good, although I'd be a bit careful about simply merging all pull requests - some of the old ones have conflicts now and might be outdated generally (including my own #133)

**Comment by gacelita** [comment]
@wuub Of course, meanwhile the org. will remain inactive
@lionandoil Yes, we'll have to be careful about that

Project owner and all people with open pull requests have been invited.

**Comment by sergiopvilar** [comment]
@JoelSanchez I'll move my PR to the new repo, thank you.

**Comment by tomschenkjr** [comment]
@JoelSanchez - no problem on my side. Mine is not a complex fork and can change the upstream repo if that does happen.

**Comment by wuub** [comment]
I slept on it and before transferring this project to an organization (and creating all the work that is associated with it) I would prefer to add some collaborators here to see if community will be able to move this project forward. 

This is also easier to arrange wrt outstanding PRs and Issues.

So please let me know if you wish to be added as an collaborator and help with cleaning up this mess :)

**Comment by gacelita** [comment]
@wuub  I'm in - I'll do what I can whenever I can. I'll start by creating a branch and trying to merge the low hanging fruit there, and merge the more difficult stuff progressively

**Comment by gacelita** [comment]
Merged half the PRs on branch `merges`. We need to check that everything works there (or, rather, that something keeps working :rofl: )

**Comment by wuub** [comment]
I'll check on my side early next week (unfortunately AFK for the whole weekend)

**Comment by fabeit** [comment]
It's crazy how forks there are, what kind of changes have people been making? I would be willing to help with the REPL R, but I am not even sure where to start.

**Comment by gacelita** [comment]
@fabeit Forget about the forks, the real work has been done through PRs, and I've merged like 20 today, in the `merges` branch. I remember some R-related changes, you could help by testing R using that branch, and reporting/fixing problems

**Comment by gacelita** [comment]
@fabeit Nope, no R changes, still it'd be great if you could do some R testing, because probably something broke

**Comment by gacelita** [comment]
@fabeit Also, lots of open issues about R (can't filter "R" properly, but they do show up): https://github.com/wuub/SublimeREPL/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20%22%20R%20%22%20

**Comment by fabeit** [comment]
@Joel I think first we should start closing all open issues.

**Comment by gacelita** [comment]
@fabeit What do you mean? Just closing them all?

**Comment by fabeit** [comment]
That we should resolve them, otherwise why would we log new issues if we can't solve the open ones?

**Comment by Kristinita** [comment]
@wuub , @JoelSanchez , maybe already possible merge branch `merges` to `master` branch?

Thanks.

**Comment by gacelita** [comment]
@Kristinita The release process keeps being under the control of the creator of the project, and anyway I don't think that releasing a version with the pull requests merged is a good idea.
I think that the best way to go is to create a new package, drop all the code that adds compatibility with ST2 (which is a real problem when releasing a new version or making changes in general) and solve issues there. I don't want to break the REPL for all existing users of this legacy (but stable) extension.

**Comment by gacelita** [comment]
@Kristinita For the time being you are free to clone the repo and checkout the merges branch, if you really need it (that's what I did).

**Comment by gacelita** [comment]
@Kristinita I can release `merges` as a new package this weekend if you want, "SublimeREPLST3" or "SublimeREPL3" or something like that.

**Comment by askielboe** [comment]
So this seems to be dead again.. Any forks you can recommend?

**Comment by fabeit** [comment]
@askielboe with all respect for the developer/s, I think this package is unusable. I have found other alternatives, like rtichoke or perhaps terminus.
